wiki/Manual.md

```diff
@@ -4780,12 +4783,14 @@ $order = $exchange->create_order ($symbol, $type, $side, $amount, $price, $param
 
 **Take Profit** / **Stop Loss** Orders which are tied to a position-opening primary order. Implemented by supplying a dictionary parameters for `stopLoss` and `takeProfit` describing each respectively.
 
-* By default stopLoss and takeProfit orders will be the same magnitude as primary order but in the opposite direction.
+* By default stopLoss and takeProfit order amounts will be the same as primary order but in the opposite direction.
 * Attached trigger orders are conditional on the primary order being executed.
-* Not supported by all exchanges.
-* Both `stopLoss` and `takeProfit` or either can be supplied, this depends on exchange.
-
-*Note: This is still under unification and is work in progress*
+* Not supported by all exchanges. To check whether stop-loss is supported, use such approach:
+```
+exchange.featureValue('swap', 'linear', 'createOrder', 'stopLoss') // if stopLoss supported
+exchange.featureValue('swap', 'linear', 'createOrder', 'stopLoss', 'price') // if limit price is supported for stoploss
+// there is also an unmaintained approach, which we no longer recommend: exchange.has['createOrderWithTakeProfitAndStopLoss'], exchange.has['createStopLossOrder'], etc..
+```
```

Comment:
I don't know what should we do here otherwise, should we list 16 variations of `createOrder-A-B-C-D` strings here to confuse users further?